presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -414,11 +306,238 @@ as a part of SQL query by appending name with the table.
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
+SQL Support
+-----------
+
+The Iceberg connector supports querying and manipulating Iceberg tables and schemas
+(databases). Here are the examples of SQL Operations supported from Presto :
+
+CREATE SCHEMA
+^^^^^^^^^^^^^^
+
+Create a new Iceberg schema named ``web`` that will store tables in an
+S3 bucket named ``my-bucket``::
+
+    CREATE SCHEMA iceberg.web
+    WITH (location = 's3://my-bucket/')
+
+CREATE TABLE
+^^^^^^^^^^^^^
+
+Create a new Iceberg table named ``page_views`` in the ``web`` schema
+that is stored using the ORC file format, partitioned by date and
+country::
+
+    CREATE TABLE iceberg.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country']
+    )
+
+Create a Iceberg table with Iceberg format version 2::
+
+    CREATE TABLE iceberg.web.page_views_v2 (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country'],
+      format_version = '2'
+    )
+
+Partition Column Transform
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Beyond selecting a particular column for partitioning, you can select a ``transform`` and partition the table by the transformed
+value of the column.
+
+Available Transforms in Presto Iceberg connector include:
+
+* ``Bucket`` (partitions data into a specified number of buckets using a hash function)
+* ``Truncate`` (partitions the table based on the truncated value of the field, and can specify the width of truncated value)
+
+Create a Iceberg table partitioned into 8 buckets of equal sized ranges::
+
+    CREATE TABLE players (
+        id int,
+        name varchar,
+        team varchar
+    )
+    WITH (
+        format = 'ORC',
+        partitioning = ARRAY['bucket(team, 8)']
+    );
+
+Create a Iceberg table partitioned by first letter of the team field::
+
+    CREATE TABLE players (
+        id int,
+        name varchar,
+        team varchar
+    )
+    WITH (
+        format = 'ORC',
+        partitioning = ARRAY['truncate(team, 1)']
+    );
+
+.. note::
+
+    ``Day``, ``Month``, ``Year``, ``Hour`` based partition column transform is not supported in Presto
```

Comment:
``Day``, ``Month``, ``Year``, ``Hour`` based partition column transform is not supported in Presto Iceberg connector
-->
``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are not supported in Presto Iceberg connector yet